src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -5088,6 +5088,19 @@
     </Field>
     <Field Name="IsActive" Type="bool" Override="true"/>
   </Node>
+  <Node Name="IgnoredDirectiveTriviaSyntax" Base="DirectiveTriviaSyntax">
+    <Kind Name="IgnoredDirectiveTrivia"/>
+    <Field Name="HashToken" Type="SyntaxToken" Override="true">
+      <Kind Name="HashToken"/>
+    </Field>
+    <Field Name="ColonToken" Type="SyntaxToken">
+      <Kind Name="ColonToken"/>
+    </Field>
+    <Field Name="EndOfDirectiveToken" Type="SyntaxToken" Override="true">
```

Comment:
probably good to add docs that say that the contents after teh `:` will be leading message on this token.  though my personal preference would be to have a dedicated token for that.